src/groups.js

```diff
@@ -88,9 +88,11 @@ function runTests(groupSchema, groupSchemaExtensions = [], configuration) {
             assert.strictEqual(response.data.schemas[0], 'urn:ietf:params:scim:schemas:core:2.0:Group', 'Response should contain the correct schema');
             assert.strictEqual(response.data.id, firstGroup.id, 'Returned group ID should match requested group ID');
             assert.ok(response.data.displayName, 'Group should contain displayName attribute');
+            // Strip content type parameters (charset, boundary) from content-type header
+            const contentType = response.headers['content-type']?.split(';')[0];
```

Comment:
I believe trimming of the result is needed, in case white space appears before the semicolon, which I believe is allowed.

Reply:
You're right, I hadn't seen it like that in the examples. However, when I dove into the specification for the `Content-Type` header I saw that it allows for whitespace before the semi-colon.

Will change when I have my laptop nearby!

Reply:
I've added a `trim()` to the groups and users tests